idmapping-rest/src/test/java/org/uniprot/api/idmapping/queue/IdMappingMessageListenerIT.java

```diff
@@ -0,0 +1,282 @@
+package org.uniprot.api.idmapping.queue;
+
```

Comment:
we need to add few more test cases here
1. Success in next attempt
2. What if job is not found in idmapping redis collection
3. empty to ids